cmd/roy/harvest_wikidata.go

```diff
@@ -21,12 +21,16 @@ import (
 	"os"
 
 	"github.com/richardlehane/siegfried/pkg/config"
-	"github.com/ross-spencer/spargo/pkg/spargo"
+	"github.com/ross-spencer/wikiprov/pkg/spargo"
 )
 
 // harvestWikidata will connect to the configured Wikidata query service
 // and save the results of the configured query to disk.
 func harvestWikidata() error {
+
+	const historyLen int = 5
+	const threads int = 10
```

Comment:
The fetch process is slow 4-6 minutes? We can achieve this with channels. Should this be configurable? (Perhaps, but it complicates the CLI args.)